### PR TITLE
PC-963: Init

### PR DESCRIPTION
**Pull request by ulfryk**

### CODE REVIEW

File: config/test-specs-for-only.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  .map(getWarnings)
+  .map(flatten)
+  .filter(warnings => warnings.length > 0)
+  .map(warnings => warnings.map(getWarningDescription).join('\n'))
+  .toEither(SUCCESS_MESSAGE)
+  .cata(toOutput(0), toOutput(1))
+  .forEach(({ message, code }) => {
+    console.log(message);
+    process.exit(code);
+  });
```
Comment (tomecko) [inline config/test-specs-for-only.js]: perhaps we can move it to some shared package?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@samwise-tech/collections",
+  "version": "0.0.1",
```
Comment (tomecko) [inline package.json]: 0.1.0 according to semver.org

Comment (ulfryk) [reply package.json]: when I'm sure it works ;)

File: config/test-specs-for-only.js (hunk 1 above)
Comment (ulfryk) [reply config/test-specs-for-only.js]: definitely - that will be my next task i suppose :)

File: package.json (hunk 2 above)
Comment (tomecko) [reply package.json]: major = 0 is already saying you're not sure ;)

**Pull request merged**